hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/SyntheticCryptoTransferApprovalMigration.java

```diff
@@ -0,0 +1,293 @@
+              where e.id = ct.sender
+            ), tokenentities as (
+              select t.*, e.key
+              from tokentransfers t
+              join entity e on e.timestamp_range @> t.consensus_timestamp::bigint
+              where e.id = t.sender
+            ), nftentities as (
+              select nft.*, e.key
+              from nfttransfers nft
+              join entity e on e.timestamp_range @> nft.consensus_timestamp::bigint
+              where e.id = nft.receiver
+            ) select * from cryptoentities
+              union all
+              select * from tokenentities
+              union all
+              select * from nftentities
+              order by consensus_timestamp asc
+            """;
+
+    private static final String UPDATE_CRYPTO_TRANSFER_SQL =
+            """
+            update crypto_transfer set is_approval = true where amount = :amount and consensus_timestamp = :consensus_timestamp and entity_id = :sender
+            """;
+    private static final String UPDATE_NFT_TRANSFER_SQL =
+            """
+            update transaction set nft_transfer = jsonb_set(nft_transfer, array[:index::text, 'is_approval'], 'true', false) where consensus_timestamp = :consensus_timestamp
+            """;
+    private static final String UPDATE_TOKEN_TRANSFER_SQL =
+            """
+            update token_transfer set is_approval = true where consensus_timestamp = :consensus_timestamp and token_id = :token_id and account_id = :sender
+            """;
+
+    private enum TRANSFER_TYPE {
+        CRYPTO_TRANSFER,
+        NFT_TRANSFER,
+        TOKEN_TRANSFER
+    }
+
+    private final MirrorProperties mirrorProperties;
+    private final NamedParameterJdbcTemplate transferJdbcTemplate;
+    private final TransferMapper transferMapper = new TransferMapper();
+
+    @Lazy
+    public SyntheticCryptoTransferApprovalMigration(
+            DBProperties dbProperties,
+            NamedParameterJdbcTemplate transferJdbcTemplate,
+            MirrorProperties mirrorProperties,
+            TransactionOperations transactionOperations) {
+        super(transferJdbcTemplate, dbProperties.getSchema(), transactionOperations);
+        this.transferJdbcTemplate = transferJdbcTemplate;
+        this.mirrorProperties = mirrorProperties;
+    }
+
+    @Override
+    public String getDescription() {
+        return "Update the is_approval value for synthetic crypto transfers";
+    }
+
+    @Override
+    protected MigrationVersion getMinimumVersion() {
+        // The version where the nft_transfer table was migrated to the transaction table
+        return MigrationVersion.fromVersion("1.81.0");
+    }
+
+    @Override
+    protected Long getInitial() {
+        return LOWER_BOUND_TIMESTAMP;
+    }
+
+    @Override
+    protected int getSuccessChecksum() {
+        return 1;
+    }
+
+    @Override
+    protected Optional<Long> migratePartial(Long lastUpperBound) {
```

Comment:
rename it to lowerBound so it's easier to understand
```suggestion
    protected Optional<Long> migratePartial(Long lowerBound) {
```

Reply:
Got it, it is renamed.